apps/studio/components/interfaces/Storage/FilesBuckets.tsx

```diff
@@ -0,0 +1,182 @@
+                        <TableRow key={bucket.id}>
+                          <TableCell>
+                            <p className="text-foreground">{bucket.name}</p>
+                          </TableCell>
+                          <TableCell>
+                            <p className="text-foreground-light">
+                              {bucket.public ? 'Public' : 'Private'}
+                            </p>
+                          </TableCell>
+                          <TableCell>
+                            <div className="flex justify-end gap-2">
+                              <Button type="default" onClick={() => {}}>
+                                View files
+                              </Button>
+                              <DropdownMenu>
+                                <DropdownMenuTrigger asChild>
+                                  <Button type="default" className="px-1" icon={<MoreVertical />} />
+                                </DropdownMenuTrigger>
+                                <DropdownMenuContent side="bottom" align="end" className="w-40">
+                                  <DropdownMenuItem
+                                    className="flex items-center space-x-2"
+                                    onClick={() => {}}
+                                  >
+                                    <Edit size={12} />
+                                    <p>Edit bucket</p>
+                                  </DropdownMenuItem>
+                                  <DropdownMenuSeparator />
+                                  <DropdownMenuItem
+                                    className="flex items-center space-x-2"
+                                    onClick={() => {}}
+                                  >
+                                    <FolderOpen size={12} />
+                                    <p>Empty bucket</p>
+                                  </DropdownMenuItem>
+
+                                  <DropdownMenuItem
+                                    className="flex items-center space-x-2"
+                                    onClick={() => {}}
+                                  >
+                                    <Trash2 size={12} />
+                                    <p>Delete bucket</p>
+                                  </DropdownMenuItem>
+                                </DropdownMenuContent>
+                              </DropdownMenu>
+                            </div>
+                          </TableCell>
+                        </TableRow>
+                      ))}
+                    </TableBody>
+                  </Table>
+                </Card>
+              </div>
+            </ScaffoldSection>
+          </div>
+        )
+    }
+  }
+
+  return (
+    <>
+      {renderTabContent()}
+      <CreateBucketModal
+        hideAnalyticsOption={true}
+        visible={showCreateBucketModal}
+        onOpenChange={setShowCreateBucketModal}
+      />
```

Comment:
hmm we'll be rendering 2 of this components if the tab is `bucket` - does the create bucket modal need to be present across all tabs?

Reply:
Hmm. Should just be the following instances:

- One `CreateBucketModal` trigger on empty state
- One `CreateBucketModal` trigger on bucket tab (1+ buckets)

The `CreateBucketModal` trigger is not available on the other tabs